### DISCUSSION TITLE
Feature Request  这里的模型怎么转换？使用？https://paddlepaddle.github.io/PaddleOCR/algorithm/table_recognition/algorithm_table_slanet.html#5-faq

### DISCUSSION THREAD

**Discussion by nissansz** [RapidAI/RapidTable#6] [Q&A]
请您详细描述想要添加的新功能或者是新特性
(Please describe in detail the new function or new feature you want to add)
这里的模型怎么转换？使用？https://paddlepaddle.github.io/PaddleOCR/algorithm/table_recognition/algorithm_table_slanet.html#5-faq

**Comment by SWHL** [comment]
建议尝试用paddle2onnx试试
